core/vm/actor/cgo/cgo_invoker.hpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_VM_ACTOR_CGO_CGO_INVOKER_HPP
+#define CPP_FILECOIN_CORE_VM_ACTOR_CGO_CGO_INVOKER_HPP
+
+#include "vm/actor/invoker.hpp"
+
+namespace fc::vm::actor::cgo {
+
+  /**
+   * Invoker implementation that invokes GoLang specs-actors using cgo
+   */
+  class CgoInvoker : public Invoker {
+   public:
+    explicit CgoInvoker(bool test_vectors);
```

Comment:
Should `InvokerImpl` have same option?

Reply:
test_vectors is used only for randomness, and randomness is provided by `Runtime`. I think it should be removed from `CgoInvoker` to `Runtime`, and `InvokerImpl` shouldn't have the option.